### PR TITLE
cmd/netcheck: better DERP latency checking output

### PR DESCRIPTION
**Pull request by lzjluzijie**
#206

### CODE REVIEW

File: cmd/tailscale/netcheck.go [hunk 1]
```diff
@@ -51,15 +51,24 @@ func runNetcheck(ctx context.Context, args []string) error {
...
-	}
-	sort.Strings(ss)
-	for _, s := range ss {
-		fmt.Printf("\t\t- %s = %v\n", s, report.DERPLatency[s])
+
+	// When DERP latency checking failed,
+	// magicsock will try to pick the DERP server that
+	// most of your other nodes are also using
+	if len(report.DERPLatency) == 0 {
+		fmt.Printf("\t* DERP latency checking failed, magicsock will pick the DERP server based on popularity\n")
```
Comment (danderson) [inline cmd/tailscale/netcheck.go:59]: Hmm, we should probably try to match the writing style of the other output of netcheck. How about:

`\t* Nearest DERP: unknown (no response to latency probes)`

It doesn't explain that DERP might still work, but I think that's okay. This is debugging output, and I think @bradfitz was going to add an explicit test that connects to DERP over TCP anyway.

Comment (lzjluzijie) [reply cmd/tailscale/netcheck.go]: Thank you!

**Comment by danderson** [comment]
One last thing, you need to add a `Signed-Off-By` line to your commit message, to satisfy DCO-bot. It's to say that you agree to https://developercertificate.org/ , which is basically 2 things:
 - You're saying that you're allowed to make this contribution (obviously true because you're @tailscale.com :) )
 - You understand that this contribution is public and open source.

It's mostly a formality for external contributors, but we require signoffs for Tailscalers as well. You can automatically insert the Signed-Off-By line in the existing commit with `git commit --amend --signoff`. For future changes, use `git commit -s` to add the signoff.

**Comment by bradfitz** [comment]
When referencing issues, say either:

`Updates #nnnn` or `Fixes #nnn` 

See https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

But ideally you put that in your git commit (not just the PR description) so it's visible in the `git log` history later.

**Comment by lzjluzijie** [comment]
@danderson @bradfitz Thank you very much. I recommitted and force-pushed to the branch. Is it OK now?

**Comment by bradfitz** [comment]
LGTM

**Pull request merged**